src/index.ts

```diff
@@ -24,11 +24,26 @@ async function run() {
     })
   )
 
-  // 3) Post comment
   if (!inputs.skipEmpty || !planIsEmpty(plan)) {
```

Comment:
This check currently results in no summary being added when no comment is posted. I think there's never a harm in adding to the summary, hence, I'd move the conditions around a little here.